This doctest relies on dictionary key order, so remove it.
--- a/docs/index.txt
+++ b/docs/index.txt
@@ -321,11 +321,6 @@
     <Element html at ...>
     >>> res.lxml.xpath('//body/div')[0].text
     'hey!'
-    >>> res = TestResponse(content_type='application/json',
-    ...                    body=b'{"a":1,"b":2}')
-    >>> res.request = TestRequest.blank('/')
-    >>> list(res.json.values())
-    [1, 2]
     >>> res = TestResponse(content_type='application/xml',
     ...                    body=b'<xml><message>hey!</message></xml>')
     >>> res.request = TestRequest.blank('/')
